core/src/zeit/content/article/tests/test_article.py

```diff
@@ -519,3 +519,15 @@ def test_article_get_searchable_text(self):
         article.body.create_item('p').text = 'Normaler Absatz'
         adapter = zope.index.text.interfaces.ISearchableText(article)
         assert adapter.getSearchableText() == ['Link', 'und mehr Text', 'Normaler Absatz']
+
+
+class WochenmarktArticles(zeit.content.article.testing.FunctionalTestCase):
+    def test_recipe_properties_are_stored(self):
+        FEATURE_TOGGLES.set('wcm_19_store_recipes_in_storage')
+        uid = 'http://xml.zeit.de/zeit-magazin/wochenmarkt/rezept'
+        article = self.repository['article'] = zeit.cms.interfaces.ICMSContent(uid)
+        with checked_out(article):
+            pass
+        article = self.repository['article']
+        self.assertEqual(len(article.recipe_titles), 2)
+        self.assertEqual(len(article.recipe_ingredients), 5)
```

Comment:
```suggestion
        self.assertEqual(['chicken-nuggets', 'bratwurst', 'brathaenchen', 'gurke', 'tomate'], sorted(article.recipe_ingredients)))
```

Wenn man eh schon sich auf die Fixture bezieht, kann mans auch gleich ganz ausschreiben, oder? Dann zeigt man auch gleich, dass es "ordentlich" übernommen wird (`[None, None,...]` wär ja auch len=5 ;)